Scripts/Player.cs

```diff
@@ -173,4 +219,19 @@ public override void _Process(float delta)
 
         onFloor = this.IsOnFloor();
     }
+
+    public float GetCurrentAir()
```

Comment:
Also, getting this error
![image](https://user-images.githubusercontent.com/11905377/35990141-135aba54-0cfb-11e8-824f-451855e8386f.png)